src/registry_schemas/schemas/agm_extension.json

```diff
@@ -12,16 +12,66 @@
             "type": "object",
             "required": [
                 "year",
-                "isFirstAgm"
+                "isFirstAgm",
+                "prevAgmRefDate",
+                "extReqForAgmYear",
+                "expireDateCurrExt",
+                "intendedAgmDate",
+                "totalApprovedExt",
+                "expireDateApprovedExt"

```

Comment:
These are not all required. Some are only required if this is not the first AGM.

I'm not sure if we should enforce conditionally required properties here or in the Legal API.

Reply:
Ok I will look into this 👍

Reply:
Don't spend more time on this because we're still waiting for answers on what we're going to do.

The AGM Extension flow (step) analysis you need to do here is the same as what is needed to design the UI itself. It would be nice to do it once (or together)... after we get the answers. Please look for another ticket in the meantime.

Reply:
Thanks for fixing.